src/screens/DetailedViewScreen.tsx

```diff
@@ -114,9 +127,9 @@ const DetailedViewScreen: React.FC<PlaceProps> = ({ setPage, placeID }: PlacePro
 						>
 							{distanceInMi ? `${distanceInMi} mi away` : ""}
 						</BodyText>
-						{Object.entries(a11yDataMap).map((attrScorePair, index) => {
+						{Object.entries(namesToFieldsMap).map((attrScorePair, index) => {
```

Comment:
it isn't really an attr / score pair, more like attr/name, so i think we should either rename this variable or:

my favorite way to use object.entries is with an array destructure like this, so you don't have entry[0] and entry[1] everywhere: `Object.entries(namesToFieldsMap).map(([attrCode, attrName], index) => {`

Reply:
ohhhh I like that!